python/lsst/daf/butler/core/datasets.py

```diff
@@ -120,10 +120,10 @@ def getNewId(cls):
 
     def __init__(self, datasetType, units):
         units = datasetType.units.conform(units)
-        super().__init__(
-            datasetType.name,
-            **{unit.__class__.__name__: unit.value for unit in units}
-        )
+#         super().__init__(
```

Comment:
Is this a temporary removal or permanent (in which case either add a comment saying when it's coming back or delete it).

Reply:
I was not sure why that code was there at all, so I wanted to check with @pschella . To me it looks like some left-over from previous versions of that code and it can be removed but again I'm not entirely sure.